db/migrate/20241215081736_add_discarded_at_to_talks.rb

```diff
@@ -0,0 +1,6 @@
+class AddDiscardedAtToTalks < ActiveRecord::Migration[8.0]
+  def change
+    add_column :talks, :discarded_at, :datetime
```

Comment:
I feel like we should call this something like `hidden_at` rather than `discarded_at`, since this feature is more about "hiding" a talk from showing up.

Reply:
discarded_at is the attribute used by the [gem discard](https://github.com/jhawthorn/discard) but we could easily do without it with a simple concern. 

For me the main question remains do we hide the talk or hide the SpeakerTalk relation 
If we hide the relation then some things we wanted to do becomes a bit more tricky 
- does appear in the event index view as grayedout (only if all SpeakerTalk are hidden)
- doesn't have a #show page (only if all SpeakerTalk are hidden)